@AH-64D Apache Longbow/Addons/fza_ah64_wca/functions/fn_update.sqf

```diff
@@ -21,10 +21,12 @@ params ["_heli"];
 
 private _onGnd      = isTouchingGround _heli;
 private _gndOrideOn = _heli getVariable "fza_ah64_gndOrideOn";
+private _battBusOn = _heli getVariable "fza_systems_battBusOn";
 
-if (!_gndOrideOn && _onGnd) then {
+if (!_gndOrideOn && _onGnd || !_battBusOn) then {
```

Comment:
None of this should function if the battery isn't on, should move this up above and use an exitWith

Reply:
thought it was simpler, to include in the if statement, that duplicating the output

Reply:
This'll work since it's the only thing being called.